spec-traits/src/main/java/ch/epfl/smithy/jsonrpc/traits/EnumKindTrait.java

```diff
@@ -0,0 +1,87 @@
+package ch.epfl.smithy.jsonrpc.traits;
```

Comment:
the Java POJOs are there to facilitate the querying of information present in the smithy model : when assembling the model, the smithy-annotations automatically get deserialised to the registered POJOs. They're not 100% necessary, but they will be helpful to implement linters too later on.